addons/mrp/static/src/components/bom_overview_line/mrp_bom_overview_line.scss

```diff
@@ -0,0 +1,3 @@
+.o_mrp_bom_overview_text_muted {
```

Comment:
You can remove this class and use `opacity-75` instead (it will make all the `td` semi-transparent instead of only the text, but it shouldn't make any difference in this case)